### DISCUSSION TITLE
[FEATURE REQUEST] better installation instruction

### DISCUSSION THREAD

**Discussion by SamuelArnold** [hlky/stable-diffusion-webui#278] [Feature suggestions]
**Is your feature request related to a problem? Please describe.**
 or an installer that installs stable diffusion and puts this and realsegan in the right place. Maybe have GPU installar and another for CPU that doesnt use CUDA

**Describe alternatives you've considered**
an alternative Better installer instructions,

**Comment by hlky** [comment]
Describe the exact problem with the install instructions. Everything that needs to be installed is very easily. The only thing you need to do is download the gfpgan and esrgan models.

>CPU

@SamuelArnold The official minimum requirements are 10GB vram, this repo works with 4GB thanks to the work by basujindal, the idea of using CPU while possible is just dumb imo, it's on a list of things to do, but it's low priority because supporting users with unsupported hardware is difficult enough as it is, the additional complaints of cpu users saying it's slow will be even more annoying. Plus there are other repos with cpu support, if that is something you need then you can use those, of course you won't get the same features as this repo.

**Reply by billythemusical** [reply]
"Describe the exact problem with the install instructions."

Well in my case, Python is not my native programming language. So it's not obvious to me why there isn't just a line: "To install, type: `pip install requirements.txt` // To launch the browser GUI, type: `python webgui.py`" ...or whatever.

**Reply by DJPatron** [reply]
@billythemusical have you seen [this](https://rentry.org/GUItard) guide? It is simple and informative, I personally used it and I can recommend it. No programming needed.

**Reply by listelin** [reply]
> Everything that needs to be installed is very easily

Yes, I think so. Thank you for your great work.
But those who come here for the first time do not know that it is easy.
Its very important to write down the simple installation steps.
